community/cypher/cypher/src/test/scala/org/neo4j/cypher/AggregationAcceptanceTest.scala

```diff
@@ -177,4 +177,21 @@ class AggregationAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerT
     //THEN
     result.toList should equal (List(Map("count(*)" -> 100)))
   }
+
+  test("aggregation around named paths works") {
+    relate(createNode(), createNode())
+
+    val a = createNode()
+    val b = createNode()
+    val c = createNode()
+    relate(a,b)
+    relate(b,c)
+    relate(createNode(), createNode())
+
+    val result = executeWithAllPlanners(
+      """match p = a-[*]->b
+        |return collect(nodes(p)) as paths, length(p) as l order by length(p)""".stripMargin)
+
+    println(result.dumpToString())
```

Comment:
It is actually testing something that is quite important - `executeWithAllPlanners` checks that RULE and COST return the same thing. But you are right. An assertion and less println sounds fine :)